### ISSUE TITLE
Suggestion - Ideas to remove money from the economy

### ISSUE THREAD

**Issue by gearsovwar** [InZidiuZ/op-framework-issue-tracker#3608]
### Assurance

I have read the `README.md`

### Summary

With the current taxes being added i would like to add a few alternative suggestions on how to remove money from the economy without being taxed.

<h2>-Premium healthcare</h2>  Option that you pay monthly/weekly for (reduces the amount of items Nancy will steal if items are stolen This could be very marginal and people would still pay into it.)

<h2>-Donation drives</h2> Donation drives for charities that will offer non tradeable items such as a trinket or a custom article of clothing or livery Could even give the winner tax exempt. (leaderboard for each donation event would help incentivize people to dono imo). 

<h2>-Titles/Custom ID's</h2> Allow custom ID's/Titles to be purchased/bid on sort of like the twitter checkmark. (really anything that people can flex they will spend money on) This could be expanded to allowing business owners having custom titles on their licenses. But the title would appear on the license. These titles could be simple Social/flex titles such as "The elite", "Top dog", "The goat" or job specific such as "CEO of xxxx" and "Best surgeon"

<h2>-Luxury penthouse</h2>They would be a very high class ownable penthouse that would be a huge money drop.)

<h2>-Custom skateboard skins</h2> Being able to create a skin for your skateboard similiar to vapes.

<h2>-Custom plates</h2>$100k+ a pop , Allows people to customise their plates on multiple vehicles (Massive cash drop). I know they offer this in the opfw store but even temporarily could be a very good way to delete money.

<h2>-Art/paintings:</h2> Paintings that can be bidded on and if won placed in a players house.

<h2>-Gang turf/block</h2> require gangs in order to have an "official" block (they can still claim anywhere) to register it.

<h2>-Furniture store</h2> Would offer limited edition or seasonal items to buy and place in your house.

<h2>-Training</h2> Allow training for various tasks such as gem stone refining that would only last x amount of days but increase the speed at which you were able to complete them.

<h2>-Tolls</h2> Sort of like taxes but atleast you can blame yourself if you go through one.

<h2>-Underground Black market shop</h2> Allow people to purchase exclusive mods and items that are very expensive and rotate daily/weekly. This could include things like car mods, crypto, Syringes/syrams that give stat buffs (kind of like steroids). Could also be done as black market auctions.

<h2>Spray paint</h2> Could be used as a money sink that would allow vandalism but has a limit on the amount and time you can use it.

<h3>Conclusion</h3> 

I know some of these suggestions might not be technically possible due to how FIvem/current scripts are implemented but hopefully it helps a little.

### Reason

It would help the economy by decreasing inflation improving server health.

**Comment by BaronVonSnergerberger** [comment]
Would also like to suggest possible versions of tickets or even straight up purchase options on log in screen for the full month of tier passes via cash. If you wanted to make them less incentivized to keep support of server unique make it so they don't have access to EDM or Imports. It's purely for train and maybe purchase of a push to apply to people not in city stuck on train. This would open up the train for more people, probably be cheaper than edm access ones in which case make it 1 to 2 weeks if you didn't want full month versions. 

The customization of plates, permanent boats are also ideas I support.

**Comment by coalaura** [comment]
> Luxury penthouse

This was suggested prior over here #3120

> Custom skateboard skins

Not possible

> Art/paintings

These would be limited to pre-made or basegame ones

> Spray paint

Not really easily doable (most scripts i found are pretty ass)

The other suggestions sound cool but a lot of them are very hard/time consuming to implement, i like em tho

**Comment by spitslave** [comment]
Can I add a suggestion here hinting at Jewerly 
I know we have necklaces in the clothing stores but this would give someone the ability to have the new business. I think people need more things to buy in the city overall, and the problem isn’t the chain robbers or the grinding but more so people literally have nothing to buy other than cars and weekly rent on house.

Jewelry Store - 
It’ll be an item you can carry in your inventory there doesn’t have to be insane customization but certain Jewelry and gems can create certain ones
Like crafting certain jewelry and when you carry it in your inventory it will be seen just like a hiking backpack you can buy from the hunting store. 

basically people who want it would have to turn in x amount of gems / stones along with some cash and these can be things they gift to SO’s or what not

**Comment by coalaura** [comment]
> Like crafting certain jewelry and when you carry it in your inventory it will be seen just like a hiking backpack you can buy from the hunting store.

This part wont work, other than that i like the idea of jewelry
